currencyfair.yaml

```diff
@@ -35,6 +35,18 @@ info:
     have its own long-lived Authorization Code that allows it to authenticate using the
     [authentication endpoint](#tag/Authentication).
 
+    # Getting Started
+    ### How do I register with CurrencyFair?
+    Please reach out to our Business Development Lead [Brian Bennett](mailto:brianbennett@currencyfair.com?subject=CurrencyFair%20API%20enquiry).
```

Comment:
I disagree with using a personal email here, it should be a distribution list. I'll pick it up with Mark